x/marker/spec/07_events.md

```diff
@@ -229,4 +229,4 @@ Type: `provenance.marker.v1.EventMarkerParamsUpdated`
 |-------------------------|-----------------------------------------------------|
 | EnableGovernance        | \{value for if governance control is enabled\}      |
 | UnrestrictedDenomRegex  | \{regex for unrestricted denom validation\}         | 
-| MaxSupply               | \{value for the max allowed supply\}                |
\ No newline at end of file
+| MaxSupply               | \{value for the max allowed supply\}                |
```

Comment:
This file has an auto-generated TOC. That script always includes a newline at the end of the file. Since I changed some of the headings in some of the spec docs, I did `make update-tocs` which resulted (among other things) in a newline being added to the end of this file.